apps/fabric-website/src/root.tsx

```diff
@@ -25,10 +25,19 @@ initializeIcons();
 
 let isProduction = process.argv.indexOf('--production') > -1;
 
+let isLocal = window.location.hostname === 'localhost' || window.location.hostname.indexOf('ngrok.io') > -1;
```

Comment:
nice

Reply:
I'd never heard of ngrok before. Signing up now with hopes that it'll make it easier to test across virtual machines without all the shared network issues.

Reply:
Its pretty awesome